packages/middleware-logger/src/logger.util.ts

```diff
@@ -0,0 +1,29 @@
+import { WriteStream } from 'fs';
+import { HttpResponse } from '@marblejs/core';
+import { Maybe, compose } from '@marblejs/core/dist/+internal';
+import { LoggerOptions } from './logger.model';
+
+export const isNotSilent = (opts: LoggerOptions) => () =>
+  !opts.silent;
+
+export const filterResponse = (opts: LoggerOptions) => (res: HttpResponse) =>
+  Maybe.of(opts.filter)
+    .map(filter => filter(res))
+    .valueOr(true);
+
+export const writeToStream = (stream: WriteStream, chunk: string) =>
+  stream.write(`${chunk}\n\n`);
+
+export const formatTime = (timeMs: number) =>
```

Comment:
I would name `timeMs` as `timeInMms`

Reply:
I would leave it as it is... BTW. why `Mms` instead of `Ms`?

Reply:
I think @pdomaleczny did a typo 😄

Reply:
Yup it is a typo 🙈